.github/workflows/build.yml

```diff
@@ -147,150 +90,39 @@ jobs:
 
   build:
     name: Build for ${{ matrix.goos }}_${{ matrix.goarch }}
-    runs-on: ${{ matrix.runson }}
     needs:
       - get-product-version
       - get-go-version
+    uses: ./.github/workflows/build-terraform-oss.yml
```

Comment:
Stuffing all build related logic in a separate workflow just to follow our convention we've used in other repos.

Reply:
Can we call this build-cli or something? We don't typically use the `oss` acronym externally, and I don't think we need to include the word terraform as we are in the terraform repo.

If you're following a convention or something that's duplicated throughout Hashicorp then you can ignore this comment 🙂

Reply:
Yep, it's just convention. We've added this because other repos have builds for oss and ent in the same repo to reduce merge conflicts between the two repos, and we just wanted to follow convention here.